apps/nextjs-website/src/helpers/chatbot.helper.tsx

```diff
@@ -103,6 +95,10 @@ export const useChatbot = (isUserAuthenticated: boolean) => {
 
   const getSession = (sessionId: string) => getChatbotQueries(sessionId);
 
+  const deleteSession = (sessionId: string) => {
+    return deleteSessionApi(sessionId);
+  };
+
```

Comment:
If there is not some manipulation of input data in this function, maybe it is not necessary?
```suggestion
```
N.B. Applay this suggestion only if the previous suggestion is applied